webpack/assets/javascripts/react_app/components/common/EmptyState/__snapshots__/EmptyState.test.js.snap

```diff
@@ -58,7 +58,8 @@ exports[`Default Empty State should render documentation when given a url 1`] =
       target="_blank"
     >
       <span
```

Comment:
As we discussed, there is no reason to have Icon here.
@orrabin can remove the Icon from the `EmptyState` please?